### PR TITLE
Nakamura/develop/readyToCss

### PR DESCRIPTION
**Pull request by kosei8**

### CODE REVIEW

File: oyster/src/components/main/TopPage/elements/CalculateSection/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { useTopPageState } from '../../hoooks/useTopPageState';
+
+const CaluculateSection = () => {
```
Comment (kazuyahirotsu) [inline oyster/src/components/main/TopPage/elements/CalculateSection/index.js:3]: typo

File: oyster/src/components/main/TopPage/elements/CalculateSection/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { useTopPageState } from '../../hoooks/useTopPageState';
```
Comment (kazuyahirotsu) [inline oyster/src/components/main/TopPage/elements/CalculateSection/index.js:1]: typo

File: oyster/src/components/main/TopPage/elements/DrawCanvasSection/index.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import { useEffect, useState } from 'react';
+import Canvas from 'react-canvas-polygons';
+
+import { useTopPageState } from '../../hoooks/useTopPageState';
```
Comment (kazuyahirotsu) [inline oyster/src/components/main/TopPage/elements/DrawCanvasSection/index.js:4]: typo

File: oyster/src/components/main/TopPage/elements/DropImageSection/index.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import { useCallback } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+import { useTopPageState } from '../../hoooks/useTopPageState';
```
Comment (kazuyahirotsu) [inline oyster/src/components/main/TopPage/elements/DropImageSection/index.js:4]: typo

**Review by kazuyahirotsu — CHANGES_REQUESTED** [review]

File: oyster/src/components/main/TopPage/elements/DropImageSection/index.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+          <input {...getInputProps()} />
+          <button>click to change image</button>
+        </div>
+      ) : (
+        <div {...getRootProps()}>
+          <input {...getInputProps()} />
+          {isDragActive ? (
+            <p>Drop the image here ...</p>
+          ) : (
+            <p>Drag &aposn drop the image here, or click to select the image</p>
```
Comment (kazuyahirotsu) [inline oyster/src/components/main/TopPage/elements/DropImageSection/index.js:47]: typo

**Comment by kosei8** [comment]
LGTM
TopPage以外も増えたらまた見直すかも構造

**Pull request merged**